patacrep/latex/syntax.py

```diff
@@ -126,20 +125,22 @@ def p_songbraces(symbols):
         else:
             symbols[0] = []
 
-    @staticmethod
-    def p_dictionary(symbols):
+    def p_dictionary(self, symbols):
         """dictionary : identifier EQUAL braces dictionary_next
                       | identifier EQUAL error dictionary_next
                       | empty
         """
+        symbols[0] = {}
         if len(symbols) == 2:
-            symbols[0] = {}
+            pass
         elif isinstance(symbols[3], ast.Expression):
-            symbols[0] = {}
             symbols[0][symbols[1]] = symbols[3]
             symbols[0].update(symbols[4])
         else:
-            raise ParsingError("Do enclose arguments between braces.")
+            self.error(
+                line=symbols.lexer.lineno,
+                message="Argument '{}' should be enclosed between braces.".format(symbols[1]),
```

Comment:
:+1: 

La première fois que j'ai vu le message `"Do enclose arguments between braces."`, j'ai lu "Do **not** enclose ..."*